pkg/template/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<!--
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<html>
+<head>
+    <title translatable="yes">PACKAGE_NAME</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <link href="PACKAGE_NAME.css" type="text/css" rel="stylesheet">
+    <script src="../base1/bundle.js"></script>
+    <script src="bundle.js"></script>
+<script>
+    require([
+        "jquery",
+        "PACKAGE_NAME/PACKAGE_NAME",
+    ], function($, PACKAGE_NAME) {
+        PACKAGE_NAME.init();
+    });
+</script>
+</head>
+
+<body>
+  <!-- CURTAIN -->
+  <div id="unsupported" hidden>
```

Comment:
Components should hide their body until they are ready to show something meaningful. The shell will display a spinner in these cases. Most components shouldn't need to use another curtain on top of that.